From 2033000a47f5d30f774ef34960bd5f8d6afa5de8 Mon Sep 17 00:00:00 2001
From: Vyacheslav Bocharov <adeep@lexina.in>
Date: Sun, 31 Oct 2021 14:52:51 +0300
Subject: [PATCH 7/7] arm64: dts: meson-gxl: update JetHub H1 dts file

Enable 23 bit of ETH PHY CNTL register: revert leds of ethernet phy.
---
 .../boot/dts/amlogic/meson-gxl-s905w-jethome-jethub-j80.dts   | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxl-s905w-jethome-jethub-j80.dts b/arch/arm64/boot/dts/amlogic/meson-gxl-s905w-jethome-jethub-j80.dts
index a5ee7ed17efa..66d92c16c133 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxl-s905w-jethome-jethub-j80.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-gxl-s905w-jethome-jethub-j80.dts
@@ -244,3 +244,7 @@ pcf8563: pcf8563@51 {
 		status = "okay";
 	};
 };
+
+&internal_mdio {
+	reg = <0xe48908ff>;
+};
-- 
2.30.2

